test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1

```diff
@@ -246,6 +246,171 @@ Describe "Basic FileSystem Provider Tests" -Tags "CI" {
     }
 }
 
+Describe "Hard link and symbolic link tests" -Tags "CI", "RequireAdminOnWindows" {
```

Comment:
Your description mentions that the -Recurse parameter switch was ignored.  But I don't see any tests to verify that it is fixed.